signaler/signaler_test.go

```diff
@@ -0,0 +1,37 @@
+//go:build !windows
+// +build !windows
```

Comment:
🚧 We should not be double bagging. We pin go to a recent version, currently 1.25 and [+build has been deprecated since 1.18](https://tip.golang.org/doc/go1.18#go-build-lines)
```suggestion
//go:build !windows
```